demo/src/app/components/+collapse/collapse-section.list.ts

```diff
@@ -29,6 +30,13 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/basic/basic.html'),
         outlet: CollapseDemoComponent
       },
+      {
+        title: 'Collapsed & expanded events listening',
+        anchor: 'events',
+        component: require('!!raw-loader?lang=typescript!./demos/events/events'),
+        html: require('!!raw-loader?lang=markup!./demos/events/events.html'),
+        outlet: CollapseDemoEventsComponent
```

Comment:
add a description field with text `Collapse directive exposes 2 events: <code>collapsed</code>, that fires when a content was hidden, and  <code>expanded</code>, that fires when a content was shown`